hbase-hadoop-compat/src/test/java/org/apache/hadoop/hbase/master/MetricsMasterWrapperStub.java

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
What is this class about w/ its hardcodings? Thanks.

Reply:
I added this class for test purpose only. It is just a wrapper stub which returns the value mapped with the particular gauge. For example, NUM_SPLIT_PROCEDURE_REQUEST_NAME will be resolved to getSplitProcedureRequestCount().